ampd/src/types.rs

```diff
@@ -55,3 +55,21 @@ impl fmt::Display for TMAddress {
         self.0.fmt(f)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use ecdsa::SigningKey;
+    use rand::rngs::OsRng;
+
+    use super::{PublicKey, TMAddress};
+
+    impl TMAddress {
+        pub fn random(prefix: &str) -> Self {
+            Self(
+                PublicKey::from(SigningKey::random(&mut OsRng).verifying_key())
+                    .account_id(prefix)
+                    .expect("failed to convert to account identifier"),
+            )
+        }
+    }
+}
```

Comment:
@cgorenflo weigh in your opinion here. The point was that I don't like including code in the production build for it that is only needed by testing. Not sure if this is the best way/location to do this though.